api/chaos/serverexperimentation/v1/serverexperimentation.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package clutch.chaos.serverexperimentation.v1;
+
+import "google/api/annotations.proto";
+import "validate/validate.proto";
+
+import "api/v1/annotations.proto";
+
+option go_package = "serverexperimentationv1";
+
+message TestConfig {
+  oneof target {
+    ClusterPairTarget cluster_pair = 1;
```

Comment:
this won't apply for all server-side tests. redline tests won't have a clusterpair, for example.

Reply:
I can take care of this in a subsequent PR.

Reply:
>I can take care of this in a subsequent PR.

Great 👍 

I think that for squeeze tests we will introduce a separate root object (other than `TestConfig`) so that the targeting in squeeze tests won't have to be the same as it is here.